content/blog/2018-08-28-birds-ocr.md

```diff
@@ -0,0 +1,411 @@
+Biodiversity Heritage Library, have a look at the [`rbhl`
+paclage](https://github.com/ropensci/rbhl), part of rOpenSci’s suite,
+that interacts with the BHL API. One can e.g. search all books by the
+same author as the one we used images from.
+
+``` r
+author <- rbhl::bhl_authorsearch("Gregory M Mathews")
+books <- rbhl::bhl_getauthortitles(creatorid = author$CreatorID)
+head(books$FullTitle)
+```
+
+    ## [1] "A manual of the birds of Australia,"                                                                                                                                                                                                                                          
+    ## [2] "A list of the birds of the Phillipian sub-region : which do not occur in Australia. "                                                                                                                                                                                         
+    ## [3] "A manual of the birds of Australia /"                                                                                                                                                                                                                                         
+    ## [4] "A list of the birds of Australia : containing the names and synonyms connected with each genus, species, and subspecies of birds found in Australia, at present known to the author /"                                                                                        
+    ## [5] "Austral avian record; a scientific journal devoted primarily to the study of the Australian avifauna."                                                                                                                                                                        
+    ## [6] "Arcana, or, The museum of natural history : containing the most recent discovered objects : embellished with coloured plates, and corresponding descriptions : with extracts relating to animals, and remarks of celebrated travellers; combining a general survey of nature."
+
+Or we could get all books whose title contains the words “birds” and
+“australia”.
+
+``` r
+australia_birds <- rbhl::bhl_booksearch(title = "birds Australia")
+head(australia_birds$FullTitle)
+```
+
+    ## [1] "Handbook to the birds of Australia. : [Supplementary material in Charles Darwin's copy]."
+    ## [2] "An introduction to The birds of Australia /"                                             
+    ## [3] "The useful birds of southern Australia : with notes on other birds /"                    
+    ## [4] "The Birds of Australia"                                                                  
+    ## [5] "The birds of Australia,"                                                                 
+    ## [6] "The birds of Australia,"
+
+And to get the OCR results of the pages of the book we used, we could
```

Comment:
can you rephrase toclarify whether you are looking for page numbers?

Reply:
I added comments in the code chunk